src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -442,4 +447,40 @@ public void parse_extraArgumentForIgnore_throwsParseException() throws ParseExce
         String input = String.format("-config %s --ignore-standalone-config true", CONFIG_FOLDER_ABSOLUTE);
         ArgsParser.parse(translateCommandline(input));
     }
+
+    @Test
+    public void parse_withTimezone_success() throws ParseException {
+        String input = String.format("-repos \"%s\" %s -timezone UTC+11", TEST_REPO_REPOSENSE, TEST_REPO_DELTA);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        Assert.assertTrue(cliArguments instanceof LocationsCliArguments);
+        Assert.assertEquals(ZoneId.of("UTC+11"), cliArguments.getZoneId());
```

Comment:
Extract the timezone String as a variable, and pass in into the input and assertion